src/module/item/weapon/index.ts

```diff
@@ -549,6 +549,8 @@ class WeaponPF2e extends PhysicalItemPF2e {
                         !(t.startsWith("thrown") && !this.isThrown) &&
                         // Finesse trait on thrown attacks with thrown melee weapons
                         !(t === "finesse" && this.isRanged) &&
+                        // Brutal trait on melee attacks
+                        !(t === "brutal" && this.isMelee) &&
```

Comment:
not sure what this is for tbh, it seems to not do anything for finesse either, but put it here because maybe there is wisdom in this.

Reply:
It makes it so the ranged attack generated for a thrown melee weapon won't have the finesse trait.

Reply:
![image](https://user-images.githubusercontent.com/1286721/183772198-747ca676-6e4d-457e-8107-680cf1e65ac5.png)

still seems to have it

Reply:
yeah but what method is that in :slightly_frowning_face: